CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ x.x.x Release notes (yyyy-MM-dd)
 ### Fixed
 * Tokens are refreshed ahead of time. If the lifetime of the token is lower than the threshold for refreshing it will cause the client to continously refresh, spamming the server with refresh requests. A lower bound of 10 seconds has been introduced. ([#2115](https://github.com/realm/realm-js/issues/2115), since v1.0.2)
 * Prevent automatic token refreshes for Realms that have been closed. Previously, these could have resulted in obscure `Unhandled session token refresh error` messages in the logs that were benign. ([#2119](https://github.com/realm/realm-js/pull/2119))
+* Aligns better with Node 10 by not using deprecated calls. ([#2107](https://github.com/realm/realm-js/issues/2107), since v2.19.0)
```

Comment:
Sorry, but without reading the issue this refers to it's not clear which problem has been fixed and how it could have affected me as a user.

Reply:
I think it should be move to `Internal` since it doesn't impact users directly. It is mostly preparing us for the situation that the call is removed in future versions of node.